editor/src/app/shared/helpers.ts

```diff
@@ -138,6 +138,13 @@ export function initSettingConfigGroup(settingGroupConfigResponse: SettingConfig
         }];
       }
 
+      // Turn ['yes', 'no'] select type into toggle input type
+      if (values.length === 2 &&
+         (values[0].value === 'yes' || values[0].value === 'no') &&
```

Comment:
I think this condition should be:
```ts
(values[0].value === 'yes' && values[1].value === 'no') || 
(values[0].value === 'no' && values[1].value === 'yes')
```
Since we need the values to be opposite of each other.